WordPress/build.gradle

```diff
@@ -450,7 +453,7 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     }
 
-    implementation 'com.github.Tenor-Inc:tenor-android-core:0.5.1'
+    implementation (name:'tenor-android-core-jetified', ext:'aar') // Jetified Tenor Gif library
```

Comment:
Tenor hasn't been updated since 2018 - it still uses the support library. I had jetify the AAR locally and include it in the repo.